taxcalc/parameters.py

```diff
@@ -112,27 +128,64 @@ def __init__(self, start_year=None, num_years=None, last_known_year=None,
             if param != "schema"
         }
 
-    def adjust(self, params_or_path, print_warnings=True, **kwargs):
+    def adjust(
+        self, params_or_path, print_warnings=True, raise_errors=True, **kwargs
+    ):
         """
-        Implements custom warning and error handling.
+        Update parameter values using a ParamTools styled adjustment.
```

Comment:
Updates the doc string with more specifics and below I add info for the method signature.